tests/check/test_check_project_metadata.py

```diff
@@ -0,0 +1,228 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Added this to the PR as well, @GraceBocek :) For some licenses you're supposed to have a short version of the license at the top of the file, with GPL 3.0 being one of them as it's copy-left, so you need to make sure that people are aware that usage should be the same license in each file. There is a short version of the license that you can use, but it's also common practice these days to use the SPDX identifier as a one liner so that the files don't have large comments at the top with the license text.

Reply:
This fixed the [failed test here](https://github.com/scribe-org/Scribe-Data/actions/runs/17192462785/job/48769937752) :)